### PR TITLE
Added Playlist Page

### PR DESCRIPTION
**Pull request by sanjitsarkar**
**In this PR,**
I have added playlist creation feature.
Preview Link [Playlist Page](https://deploy-preview-7--zentube.netlify.app/)
**Things to check**
- [ ] PlaylistModalContext
- [ ] PlaylistContext
- [ ] AddToPlaylistModal
- [ ] VideoCard

Things not to check
- [ ] Add  video to Playlist

### CODE REVIEW

File: src/components/Header/index.jsx [hunk 1]
```diff
@@ -26,16 +29,39 @@ const Header = () => {
...
+            </Link>
+          </li>
+        )}
+        {isLoggedIn && (
+          <>
+            <li>
+              <Link to="/">
+                <img
+                  src="https://bestprofilepictures.com/wp-content/uploads/2021/04/Cool-Profile-Picture.jpg"
+                  alt="user"
```
Comment (JowelTisso) [inline src/components/Header/index.jsx]: You can store these links in a constant

File: src/context/AuthContext.jsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+          firstName: signupCred.firstName,
+          lastName: signupCred.lastName,
+          email: signupCred.email,
+          password: signupCred.password,
+        })
+      )
+      .then((res) => {
+        setToast({
+          show: true,
+          content: `Welcome, ${res.data.createdUser.firstName}`,
```
Comment (JowelTisso) [inline src/context/AuthContext.jsx]: Try using async await it will make it more readable and clean

File: src/context/HistoryContext.jsx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        type: ACTION_TYPE_SUCCESS,
+        payload: [],
+      });
+      return;
+    }
+
+    axios
+      .post(
+        "/api/user/history",
+        JSON.stringify({
```
Comment (JowelTisso) [inline src/context/HistoryContext.jsx]: You can store these api in constants and reuse them where required

**Review by JowelTisso — APPROVED** [review]
* Your PR is too big
* Try keeping your PR small
* Rest is good
* Clean code
* UI looks good
* Provide test credentials for easy login
* Good video player description UI
* Good css classes
* Looks good to merge

**Review by sanu29 — APPROVED** [review]
All good to merge...

**Pull request merged**